tripal_chado/src/Plugin/Field/FieldType/ChadoPropertyTypeDefault.php

```diff
@@ -54,14 +54,8 @@ public static function tripalTypes($field_definition) {
     // Create variables for easy access to settings.
     $entity_type_id = $field_definition->getTargetEntityTypeId();
     $settings = $field_definition->getSetting('storage_plugin_settings');
-    $base_table = $settings['base_table'];
-    $prop_table = $settings['prop_table'];
-
-    // If we don't have a base table then we're not ready to specify the
-    // properties for this field.
-    if (!$base_table) {
-      return;
-    }
```

Comment:
You took this out. I will continue tomorrow, but I am suspicious of this...

Reply:
I'm not sure why I took that out.  I recall I needed to but I can't reproduce why. Putting it back in fixes the error you mention but doesn't cause a problem elsewhere.  So, back it goes.